akka-actor-typed/src/main/resources/reference.conf

```diff
@@ -76,6 +76,9 @@ akka.reliable-delivery {
       # The ProducerController retries requests to the durable queue this
       # number of times before failing.
       retry-attempts = 10
+
+      # The ProducerController uses this interval as a delay before sending the first message.
```

Comment:
it's not a delay before sending first message
```
# The ProducerController retries sending the first message with this interval
# until it has been confirmed.
```

Reply:
Oh, sorry. I understood it wrong. Fixed